### DISCUSSION TITLE
Pnp Modern Web Part Search Results from Microsoft List no showing Rich Text or formatting

### DISCUSSION THREAD

**Discussion by 19STELLA84** [microsoft-search/pnp-modern-search#3757] [General]
Hi Guys,

I'm building a Intranet for our Quality control. I'm working with Microsoft Lists to list the issues. So added columns like 'Ítem', 'Description', risk Number', 'Solutions' etc.. It's just the best way to work for us. To make this attractive and readabe for users on the Intranet, I created a page in Sharepoint for each item. In each page I'm using the Search Result Web Part to display a particulair item from the column. All that working perfect with fetching the data. So everything fine with the Crawled and Mapped properties (on site level, not tenant). I used the RefinableString(s) available. My problem is in the columns 'Multiple lines of text' and using Rich Text and not showing that formatting on the page in the PnP Search Result.

**So my list is Rich Text**
<img width="169" alt="ListRT" src="https://github.com/microsoft-search/pnp-modern-search/assets/167984030/f82c2a5a-5f8e-4361-9d4c-4ffedbd9c4bd">

**And the Result on the page:**
<img width="505" alt="PnPSearchReultsPage" src="https://github.com/microsoft-search/pnp-modern-search/assets/167984030/e85218cb-9581-47c9-9ae1-2d69c81ff3f2">

For the web part I tried all the options and adjusting them with Json, not working (is funny though I managed more advanced layout options with trying Json, but this simple issue, just breaking my head over it) Tried on Page 2/4 - Handlebars/HTML options aswell as Adaptive Cards. For now I left it on Handlebars/HTML --> Custom. And then just used 'Manage Columns' / just one handlebar I created for the item.

Any other (custom) option is fine for me, as long as it is showing the formatting from the list.

I did read and tried the one with the 3 muchstaches, not working for me (https://github.com/microsoft-search/pnp-modern-search/issues/533) and of course had so many videos and forums for weeks now and no solution yet.

Hope you guys can help! Thanks so much in advance!

**Comment by kasperbolarsen** [comment]
I haven't worked with rich text in search, but this old classic search question might be useful,  https://sharepoint.stackexchange.com/questions/138706/rich-text-column-content-in-display-template-for-search-results-webpart

**Comment by 19STELLA84** [comment]
> I haven't worked with rich text in search, but this old classic search question might be useful,  https://sharepoint.stackexchange.com/questions/138706/rich-text-column-content-in-display-template-for-search-results-webpart

Thanx Kasper for replying, I really appreciate it! Tried this today, stuck now on that the new column doest get crawled or an automatic managed propertie. Reindexed list and site but not showing. I know sometimes it takes a bit longer to suddenly show up in the Search Scheme, I will update later! Thanx again!

**Reply by atagadtad** [reply]
Hi @19STELLA84! Did you ever manage to solve this? I'm coming across this same problem myself. 

Thanks!
